elevate/posix.py

```diff
@@ -26,7 +26,7 @@ def elevate(show_console=True, graphical=True):
     if os.getuid() == 0:
         return
 
-    args = [sys.executable] + sys.argv
+    args = [sys.executable, os.path.realpath(sys.argv[0]), *sys.argv[1:]]
```

Comment:
I don't think this syntax (`*` expansion in a list literal) is supported in Python 2. 

Could we also switch to `os.path.abspath`? I don't think we should resolve symlinks unless there's a good reason.

Suggest something like:

```
args = [sys.executable, os.path.abspath(sys.argv[0])]
args += sys.argv[1:]
```

Reply:
Ah, I overlooked Python 2 compat was already supported. We'll leave link traversal up to the Python interpreter then